packages/graphql/src/schemaTypes/objectTypes/gql-CachedViewer.ts

```diff
@@ -0,0 +1,19 @@
+import dedent from 'dedent'
+import { objectType } from 'nexus'
+
+export const CachedViewer = objectType({
+  name: 'CachedViewer',
```

Comment:
Currently not in use, was thinking it'd be useful as an eager response of the currently logged in user data to prevent a jump in the header if the `cloudViewer` is delayed due to network / if the user is offline.

Reply:
:grimacing: there is already a lot of unused code, it makes it difficult to search through the codebase, any way we can not introduce this file?

Reply:
If we go with the zero delay then we'll probably want to use this. Will revisit before it gets merged.